scripts/data/local-db.sql

```diff
@@ -165,16 +165,26 @@ e7946bd7-2e74-4263-9e97-8c326388a707	ce3405a0-3399-4e3a-88d7-3cfc613d2905	{YES_S
+f6357d27-d241-4252-a48e-240d33d3ee30	95399697-8c37-4225-b09d-7ca4fd0ad2b0	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	MINT	2022-06-06 13:10:45.731559+00	\N	\N	READY
+\.
+
+
+--
+-- Data for Name: plan_payments; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.plan_payments (id, model_plan_id, funding_source, funding_source_trust_fund_description, funding_source_other_description, funding_source_note, funding_source_r, funding_source_r_trust_fund_description, funding_source_r_other_description, funding_source_r_note, pay_recipients, pay_recipients_other_specification, pay_recipients_note, pay_type, pay_type_note, pay_claims, pay_claims_other_description, should_any_providers_excluded_ffs_systems, should_any_providers_excluded_ffs_systems_note, changes_medicare_physician_fee_schedule, changes_medicare_physician_fee_schedule_note, affects_medicare_secondary_payer_claims, affects_medicare_secondary_payer_claims_explanation, affects_medicare_secondary_payer_claims_note, pay_model_differentiation, creating_dependencies_between_services, creating_dependencies_between_services_note, needs_claims_data_collection, needs_claims_data_collection_note, providing_third_party_file, is_contractor_aware_test_data_requirements, beneficiary_cost_sharing_level_and_handling, waive_beneficiary_cost_sharing_for_any_services, waive_beneficiary_cost_sharing_service_specification, waiver_only_applies_part_of_payment, waive_beneficiary_cost_sharing_note, non_claims_payments, non_claims_payments_other_description, payment_calculation_owner, number_payments_per_pay_cycle, number_payments_per_pay_cycle_notes, shared_systems_involved_additional_claim_payment, shared_systems_involved_additional_claim_payment_note, planning_to_use_innovation_payment_contractor, planning_to_use_innovation_payment_contractor_note, funding_structure_description, expected_calculation_complexity_level, expected_calculation_complexity_level_note, can_participants_select_between_payment_mechanisms, can_participants_select_between_payment_mechanisms_description, can_participants_select_between_payment_mechanisms_note, anticipated_payment_frequency, anticipated_payment_frequency_other_description, anticipated_payment_frequency_notes, will_recover_payments, will_recover_payments_notes, anticipate_reconciling_payments_retrospectively, anticipate_reconciling_payments_retrospectively_notes, payment_start_date, payment_start_date_notes, created_by, created_dts, modified_by, modified_dts, status) FROM stdin;
+a18d2f8c-e5cb-4c5d-b21c-3554dbec9c4d	6e224030-09d5-46f7-ad04-4bb851b36eab	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	\N	FAKE	2022-07-05 13:44:44.380055+00	\N	\N	READY
```

Comment:
@OddTomBrooks Can you add an entry for each model plan here? I think the rows could each just be duplicates of this (all `NULL`, minus the `created_by` and `created_dts`).

The API will complain if you try and request a model plan + it's payments and it doesn't exist (rather than existing and having `NULL` entries)

Reply:
Yes I am working on this now :)

Reply:
@OddTomBrooks & @ClayBenson94 , I just reviewed, I saw you guys just talked about this.

I think that we should at least have one row that has data in it. The way it is, it doesn't throw an exception, but I think having data would speed up front-end development, and match what's been done for the other sections. Thoughts?